client_with_ssl.cpp

```diff
@@ -1,84 +1,176 @@
+// Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 #include <string>
 #include <hazelcast/client/hazelcast_client.h>
 
 /**
-* This is boilerplate application that configures client to connect Hazelcast
-* Cloud cluster.
-* <p>
-* See: <a href="https://docs.hazelcast.com/cloud/cpp-client">https://docs.hazelcast.com/cloud/cpp-client</a>
+* A sample application that configures a client to connect to an Hazelcast Viridian cluster
+* over TLS, and to then insert and fetch data with SQL, thus testing that the connection to
+* the Hazelcast Viridian cluster is successful.
+*
+* See: https://docs.hazelcast.com/cloud/get-started
 */
 
 void map_example(hazelcast::client::hazelcast_client client);
```

Comment:
these function declarations should be deleted;
void map_example(hazelcast::client::hazelcast_client client);
void non_stop_map_example(hazelcast::client::hazelcast_client client);

These function declarations should be added.
void create_mapping(hazelcast::client::hazelcast_client client);
void insert_cities(hazelcast::client::hazelcast_client client);